main.tf

```diff
@@ -118,3 +118,31 @@ module "ansible" {
   envs      = ["host=${aws_eip.default.public_ip}"]
   playbook  = "${var.ansible_playbook}"
 }
+
+# Restart dead or hung instance
+
+resource "null_resource" "check_alarm" {
+  triggers = {
+    action = "arn:aws:swf:${var.aws_region}:${var.aws_account_id}:${var.default_alarm_action}"
```

Comment:
You can use a data source to retrieve `aws_region` and `aws_account_id`. No need for additional inputs.